src/components/Router.js

```diff
@@ -52,6 +53,24 @@ export default class Router extends Component {
     initNavPaths()
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    const { pathname, search } = this.state
+    if (prevState.pathname === pathname) return
+
+    const handler = Nav.findHandler(pathname)
+    if (!handler) return
+
+    if (handler.title) {
```

Comment:
nit: You could tweak the logic here to just be `if (handler && handler.title) {`, which would make the 404 page use the default `Saturn`, rather than retaining the previous title.

Reply:
Good catch, I forgot about 404.